projects/1.16/assets/occultism/occultism/patchouli_books/dictionary_of_spirits/zh_cn/entries/getting_started/first_ritual.json

```
@@ -10,7 +10,12 @@
     {
       "type": "spotlight",
       "item": "occultism:chalk_white",
-      "text": "第一步，在熔炉中烧炼异界石，并且在$(thing)灵火$()中灼烧异界原木。将产出物合成为含杂白色粉笔，再将其投入$(thing)灵火$()中再次进行净化。净化后的粉笔可用于画出组成五芒星的仪式符文。可选的五芒星可以在对应章节中找到。要了解更多关于粉笔的信息，请参阅$(l:pentacles/chalks)粉笔类型$(/l)一节。"},
+      "text": "第一步，在熔炉中烧炼异界石，并且在$(thing)灵火$()中灼烧异界原木。将产出物合成为含杂白色粉笔，再将其投入$(thing)灵火$()中再次进行净化。净化后的粉笔可用于画出组成五芒星的仪式符文。可选的五芒星可以在对应章节中找到。"
```

```suggestion
      "text": "第一步，在熔炉中烧炼异界石，并且在$(thing)灵火$()中灼烧异界原木。将产生的灰烬合成为含杂白色粉笔，接着将其再次投入$(thing)灵火$()中净化。净化后的粉笔可用于画出组成五芒星的仪式符文。可选的五芒星可以在对应章节中找到。"
```
这个木头是作燃料吗？如果是的话建议“灼烧”→“燃烧”或者“用异界原木作为灵火的燃料”

---

并不是燃料